demos/datepicker/datepicker-fr.js

```diff
@@ -6,7 +6,7 @@
 	if ( typeof define === "function" && define.amd ) {
 
 		// AMD. Register as an anonymous module.
-		define([ "../jquery.ui.datepicker" ], factory );
+		define([ "datepicker" ], factory );
```

Comment:
I updated my PR on jqueryui.com to handle this as well as this PR to include removal of these from the demo and using the ones in the `ui/i18n` folder